dbux-graph-host/src/components/GraphRoot.js

```diff
@@ -15,13 +15,21 @@ export class RunNodeMap {
     this._all.set(this.makeKey(applicationId, runId), node);
   }
 
+  delete(applicationId, runId) {
```

Comment:
Name change required:
The name `GraphRoot.delete` suggests that it deletes the root itself. However, it seems to delete some child node.